wallets/provider-trezor/src/signers/ethereum.ts

```diff
@@ -0,0 +1,61 @@
+import type { EvmTransaction } from 'rango-types/lib/api/main';
+
+import TrezorConnect from '@trezor/connect-web';
+import { JsonRpcProvider } from 'ethers';
+import { type GenericSigner } from 'rango-types';
+
+import { ETH_BIP32_PATH } from '../helpers';
+
+export const RPC_PROVIDER_URL = 'https://rpc.ankr.com/eth';
```

Comment:
Considering that this line is duplicate with a line in ledger-provider, I think you can move it to "@rango-dev/wallets-shared" package.